README.md

```diff
@@ -1,141 +1,17 @@
-# Getting started with Liberty on Bluemix
-By following this guide, you'll set up a development environment, deploy an app locally and on Bluemix, and integrate a Bluemix database service in your app.
+
+# Liberty on Bluemix getting started application
+The Bluemix Getting started tutorial for Liberty uses this sample application to provide you with a sample workflow for working with any Liberty app on Bluemix; you set up a development environment, deploy an app locally and on Bluemix, and integrate a Bluemix database service in your app.
+
+The Liberty app uses Liberty's [`mongodb-2.0` feature](https://www.ibm.com/support/knowledgecenter/en/SSEQTP_liberty/com.ibm.websphere.wlp.doc/ae/twlp_mongodb_create.html) and [MongoDB Java driver](https://mongodb.github.io/mongo-java-driver/) to add information to a database and then return information from a database to the UI.
```

Comment:
I'm sorry, I mixed up sample apps. This one uses https://github.com/cloudant/java-cloudant and not the Liberty feature or MongoDB Java Driver.